### PR TITLE
cip-2 & cip-3 voting begins

### PR DESCRIPTION
**Pull request by mattjoyal**
also typo updated in text

### CODE REVIEW

**Comment by mattjoyal** [comment]
Updated CIP-2/3 to `Accepted`. CIP-4 set to `voting`. Ready to merge after approval.

**Comment by mattjoyal** [comment]
cip-4 accepted. all CIP changes have been made, ready to merge.

**Review by jim-stanczak — APPROVED** [review]

**Pull request merged**
